var/spack/repos/builtin/packages/grace/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Grace(AutotoolsPackage):
+    """Grace is a WYSIWYG 2D plotting tool for the X Window System and M*tif."""
+
+    homepage = "http://plasma-gate.weizmann.ac.il/Grace"
+    url      = "ftp://ftp.fu-berlin.de/unix/graphics/grace/src/grace5/grace-5.1.25.tar.gz"
```

Comment:
Curious.  Why are you using the mirror from Germany instead of the g FTP site?

Reply:
I haven't been able to access `ftp://plasma-gate.weizmann.ac.il/pub/grace/` in the last months. Does it work for you?

Reply:
Fair enough.

Reply:
Can you add a comment above line 13 to explain use of the mirror?

Reply:
Done.